routers/auth_router.js

```diff
@@ -12,6 +12,7 @@ router
         const user = new users({
             login: req.query.login,
             password: req.query.password,
+            isAdmin: false
```

Comment:
isAdmin: false should be set as default. No need to path it every time while creating user. Use "default" in Schema